### ISSUE TITLE
Re-used layout (or re-used elements in this case) not allowed

### ISSUE THREAD

**Issue by PySimpleGUI** [Cynwell/Image-Labeling-Helper#1]
Your label_helper.py code now generates an error/warning.  The problem is the attempted re-use of  your Button Elements.  ALL parts of a layout must be completely unused when you create a window.  By creating the Next and Prev buttons ahead of time you only get to use them once.  

To get around this, create the buttons from scratch in your `selectFilesLayout` function just like you do the Input element.

You were cheating before and getting away with it.  I've added code to make sure people don't get away with cheating any more :-)

I like that you're setting an excellent example by creating your layouts in functions.  It's the "Right way" to do it.

**Comment by Cynwell** [comment]
Ok I'll work on this problem :)

**Comment by Cynwell** [comment]
I fixed the issue by calling functions to return buttons, such that new elements are created upon being used.

Example:

def nextBtn():
    return sg.Button("Next")

This function returns a new Button element every time it is being called, which solved the issue.

**Comment by PySimpleGUI** [comment]
Perfect solution and also perfect explanation!

It's the "PySimpleGUI Golden Rule" after all.  You don't get very many "Golden Rules" to use in a project, so I had to choose pretty carefully.

https://pysimplegui.readthedocs.io/en/latest/#the-golden-rule-of-window-layouts

This technique of returning Elements from a function call is called "User Defined Elements".

https://pysimplegui.readthedocs.io/en/latest/#user-defined-elements-compound-elements

I just used them in an example of how to resize images and place onto buttons that's posted in this Issue's thread:

https://github.com/PySimpleGUI/PySimpleGUI/issues/1984

You've got the point.  I'm glad to see you using the package 😃
